src/Symfony/Component/Messenger/DependencyInjection/MessengerPass.php

```diff
@@ -30,20 +30,9 @@
  */
 class MessengerPass implements CompilerPassInterface
 {
-    private $handlerTag;
-    private $busTag;
-    private $receiverTag;
-
-    public function __construct(string $handlerTag = 'messenger.message_handler', string $busTag = 'messenger.bus', string $receiverTag = 'messenger.receiver')
-    {
-        if (0 < \func_num_args()) {
-            trigger_deprecation('symfony/messenger', '5.3', 'Configuring "%s" is deprecated.', __CLASS__);
-        }
-
-        $this->handlerTag = $handlerTag;
-        $this->busTag = $busTag;
-        $this->receiverTag = $receiverTag;
-    }
+    private $handlerTag = 'messenger.message_handler';
+    private $busTag = 'messenger.bus';
+    private $receiverTag = 'messenger.receiver';
```

Comment:
Let's inline the properties as well. https://github.com/symfony/symfony/pull/40468#issuecomment-845791510

Reply:
Sure